TramsDataApi/DatabaseModels/TramsDbContext.cs

```diff
@@ -17,6 +17,7 @@ public TramsDbContext(DbContextOptions<TramsDbContext> options)
         public virtual DbSet<AcademyTransferProjects> AcademyTransferProjects { get; set; }
         public virtual DbSet<TransferringAcademies> TransferringAcademies { get; set; }
         public virtual DbSet<AcademyConversionProject> AcademyConversionProjects { get; set; }
+        public virtual DbSet<ProjectNote> ProjectNotes { get; set; }
```

Comment:
Could this be `AcademyConversionProjectNotes` in case other services also need project notes